bee-api/bee-rest-api/src/types/dtos.rs

```diff
@@ -308,24 +320,21 @@ pub struct TreasuryInputDto {
 pub enum OutputDto {
     Simple(SimpleOutputDto),
```

Comment:
This needs to be removed

Reply:
Not sure I understand, don´t we have a SimpleOutput anymore?

Reply:
Not sure I understand, don´t we have a SimpleOutput anymore?

Reply:
No, the extended output can do the same, so it will be removed to reduce complexity https://github.com/iotaledger/bee/issues/942